controllers/configmap_controller.go

```diff
@@ -600,17 +600,19 @@ func (r *ConfigMapReconciler) ensureWICDRoleBinding(ctx context.Context) error {
 			return fmt.Errorf("unable to delete RoleBinding %s/%s: %w", r.watchNamespace, wicdRBACResourceName, err)
 		}
 		r.log.Info("Deleted malformed resource", "RoleBinding",
-			kubeTypes.NamespacedName{Namespace: r.watchNamespace, Name: existingRB.Name},
+			kubeTypes.NamespacedName{Namespace: existingRB.Namespace, Name: existingRB.Name},
 			"RoleRef", existingRB.RoleRef.Name, "Subjects", existingRB.Subjects)
 	}
 	// create proper resource if it does not exist
-	_, err = r.k8sclientset.RbacV1().RoleBindings(r.watchNamespace).Create(ctx, expectedRB,
+	createdRB, err := r.k8sclientset.RbacV1().RoleBindings(r.watchNamespace).Create(ctx, expectedRB,
 		meta.CreateOptions{})
 	if err != nil {
 		return fmt.Errorf("unable to create RoleBinding %s/%s: %w", r.watchNamespace, wicdRBACResourceName, err)
 	}
 	r.log.Info("Created resource", "RoleBinding",
-		kubeTypes.NamespacedName{Namespace: r.watchNamespace, Name: expectedRB.Name})
+		kubeTypes.NamespacedName{Namespace: createdRB.Namespace, Name: createdRB.Name},
```

Comment:
wouldn't this be same as expectedRB?

Reply:
Yes, with namespaced resources isn't a big deal. I made this change to better align with the ClusterRoleBinding log messages.